Frameworks/UIKit/UITabBarButton.mm

```diff
@@ -126,15 +126,14 @@ - (void)drawRect:(CGRect)pos {
         id font = [UIFont defaultFont];
         size = [title sizeWithFont:font constrainedToSize:CGSizeMake(0.0f, 0.0f) lineBreakMode:UILineBreakModeClip];
 
-        CGRect textRect;
-        textRect.origin.y = rect.size.height - size.height;
+        CGRect textRect{};
```

Comment:
If you're gonna do this, y'might'swell fold the following statements into the initializer.